pkg/retry/retry_test.go

```diff
@@ -92,7 +92,7 @@ func (tr *testTime) Execute(ctx context.Context) error {
 	}
 	tr.prevExecution = now
 
-	return errors.New("some error")
+	return retry.RetryableError{Err: errors.New("retryable error")}
 }
```

Comment:
This is should be a `RetryableError` because well we want to test retries. 

Despite it not being a retryable error, the test is still passing in `main` branch, because the previous version of assert library was not doing a `Len()` check in `assert.InEpsilonSlice()`, so any empty slice would always pass the assertion.

```go
assert.InEpsilonSlice(
    t,
    [1, 2, 3, 4], // expected
    [], // actual
) // This would not fail the test! 
``` 

And since the error was not retryable, `tr.waitTimes` was always an empty slice.